core/templates/dev/head/i18n.js

```diff
@@ -75,8 +75,9 @@ oppia.config([
   var availableLanguageKeys = [];
   var availableLanguageKeysMap = {};
   for (var prop in GLOBALS.SUPPORTED_SITE_LANGUAGES) {
-    availableLanguageKeys.push(prop);
-    availableLanguageKeysMap[prop + '*'] = prop;
+    availableLanguageKeys.push(GLOBALS.SUPPORTED_SITE_LANGUAGES[prop].id);
```

Comment:
Hmm... I tried this but I am not able to solve the problem

I tried setting GLOBALS.SUPPORTED_SITE_LANGUAGES to a value but still 457 tests were failing.

I also tried something like this `beforeEach(module('oppia', GLOBALS.SUPPORTED_SITE_LANGUAGES));` but even this does not solve the problem.

Also I checked that in `for (var prop in GLOBALS.SUPPORTED_SITE_LANGUAGES)` when I log the value of prop it comes out to be the index value of the elements. I don't understand why this might be the reason that the above code works.

Another question, a total of 457 tests fail. Modifying the beforeEach() block of every test would be quite tedious right?